src/v2/schema/message.js

```diff
@@ -0,0 +1,32 @@
+import { gql } from 'apollo-server-express';
+
+export const typeDef = gql`
+  type Message {
+    id: String
+    kind: String
+    description: String
+  }
```

Comment:
Document the schema.
```suggestion
  type Message {
    # Unique identifier for each message.
    id: String
    # Describes the type of message. Expected values are: information, warning, error.
    kind: String
    # Message text.
    description: String
  }
```